jupyterlab_gitplus/handlers.py

```diff
@@ -158,7 +163,7 @@ def post(self):
 
 
 class ServerConfigHandler(IPythonHandler):
-    def initialize(self, github_token, server_root_dir):
+    def initialize(self, server_root_dir, **kwargs):
```

Comment:
it'd be consistent to simply accept the "context" dict & fetch the required values from it (like below). Right now  we're passing the same values to each handler but accepting those in each handler slightly differently.

```
class ServerConfigHandler(IPythonHandler):
    def initialize(self, context):
        self.server_root_dir = context['server_root_dir']
```

```
class CommitHandler(IPythonHandler):
    def initialize(self, context):
        self.github_token = context['github_token']
        self.reviewnb_endpoint = context['reviewnb_endpoint']
```        
        
 ```
 class PullRequestHandler(IPythonHandler):
    def initialize(self, context):
        self.github_token = context['github_token']
        self.reviewnb_endpoint = context['reviewnb_endpoint']
```

Reply:
I've updated the PR to do it this way, but I'm not sure it's cleaner. 

Note how in the init we need to do `{"context": context}` because the last arg of `web_app.add_handlers` is expected to be kwargs.